### DISCUSSION TITLE
[Question]: Compatibility List, Features and Recommended Hardware.

### DISCUSSION THREAD

**Discussion by Hexadecimalism** [blakeblackshear/frigate#10232] [Ask A Question]
### What is your question:

I did a lot of searching which led me to frigate. It seems to do everything I'd like it to do while self-hosting it.  Unfortunately, the recommended hardware list is... lacking. I'm now in the never ending hell of trying to find a camera that is compatible in the same ways the recommended ones are Loryta(Dahua) T5442TM-AS-LED, Loryta(Dahua) IPC-T5442TM-AS and Amcrest IP5M-T1179EW-28MM. Everything I've found that is similar is basically a sub-brand/rebrand of Dahua and Hikvision cameras. The closest thing I've found to a compatibility list is the [ONVIF PTZ camera recommendations](https://docs.frigate.video/configuration/cameras#onvif-ptz-camera-recommendations).
Security can be an issue for me, and there has been a history of security problems with both brands, but I'm less worried about issues like that since I can firewall the devices. The bigger issue for me is due to the fact that I **personally** do not want to support brands owned by the Chinese state and leveraged in the [persecution of Uyghur](https://en.wikipedia.org/wiki/Persecution_of_Uyghurs_in_China) people. Used versions (a method which would not give said companies value) don't seem to exist regionally for me on ebay either. I'd consider reolink cameras (a brand from Hong Kong). Unfortunately, there doesn't seem to be a shortage of [issues](https://github.com/blakeblackshear/frigate/issues?q=reolink) regarding them.
A wiki of devices, compatibility and issues would be really great to have. In the interim, if anyone could recommend a fully compatible camera that might meet the ethical imperatives I'm trying to follow, please do.

[edit]: I should probably list what I need from a camera. I currently use Google Nest outdoor cameras and they have worked phenomenally since 2018. They are getting long in the tooth and I expect them to start dying sometime soon. In the mean time, I'm in a situation where I need a new camera for a new location. Basically for the far end of my property to keep an eye on the property, our chickens and local wildlife (and possibly vermin). I didn't want another nest camera consuming upstream, especially now that I have a home server consisting of storage and three clusters. When my nest cameras start going I'll be replacing them with self hosted options. 
My home network is Unifi based, and while I could consider Unify cameras, I've been less than impressed with with my network on a few occasions. On those occasions it locks up, and completely drops everything, forcing a router reboot, and requiring readoption of my other ui network devices, some of which fail and then have to be factory reset before they can be adopted again. I'm not very interested in the possibility of having to factory reset my cameras if that ever happens again, especially when they'll be placed in hard to reach places.

Anyway, I need these features from camera recommendations:

- POE
- Excellent NV
- Preferably under 200 (GBP or USD) though above is not a deal breaker
- 1080p though 2K is highly preferred
- 3MP though 5MP is highly preferred

**Comment by Hueristic** [comment]
Same building a system, subbed.

**Reply by Hexadecimalism** [reply]
> Same building a system, subbed.

I'll be honest with you, I was kind of disappointed to get no replies, and it seems to be an issue not many rate as an important issue. I personally attempt not to create value for companies I disagree with ethically. Unfortunate not to have brand new devices, for companies I'm willing to support.
The best solution I found was to buy pre-owned versions of the cameras (or their latest models) listed in the documentation, that way the company gets no additional value from you. I got them from outside my region and had a friend that was visiting transport them from the US. I also firewalled them so they have no access to the internet, or anywhere outside the vlan I put them on (though I'm still paranoid of them possibly being a security risk).

In retrospect, I should have just bought cameras from ubiquiti since they solved the networking problem I was having with an update to unifi OS, unfortunately that only happened this month.

Still, it would be nice to get recommendations in the thread for future people interested devices along the specifications I've listed.

**Reply by Hueristic** [reply]
I was reading some threads and saw a recommendation for  "Compatible with Hik-vision NVR Through ONVlF Protocol", so I ordered this to test with.

https://www.amazon.com/gp/product/B09Z1R3YR9?psc=1

**Reply by Hexadecimalism** [reply]
> I was reading some threads and saw a recommendation for "Compatible with Hik-vision NVR Through ONVlF Protocol", so I ordered this to test with.
> 
> https://www.amazon.com/gp/product/B09Z1R3YR9?psc=1

Best of luck. I remember Real HD showing up on servethehome, but I know nothing else about it as a brand.

**Reply by Hueristic** [reply]
All I saw was a swith review on that site.
I have a ton of switches so will order one of these.
https://www.aliexpress.us/item/3256801182387009.html?algo_exp_id=1772a038-5bcd-47da-80f0-310141efe372-18&pdp_npi=4%40dis%21USD%214.77%214.77%21%21%214.77%214.77%21%402101fb1917212416630992575e814c%2112000024314865388%21sea%21US%21120064114%21X&curPageLogUid=Aaprdo5Kn7os&utparam-url=scene%3Asearch%7Cquery_from%3A

**Reply by btdeviant** [reply]
> https://www.amazon.com/gp/product/B09Z1R3YR9?psc=1

How'd your tests go?
